CM.Voice.VoiceApi.Sdk/VoiceApiClient.cs

```diff
@@ -64,23 +63,22 @@ public async Task<VoiceApiResult<CallQueuedEvent>> SendAsync(BaseAppInstruction
         }
 
         [Obsolete("See SendAsync")]
-        public Task<VoiceApiResult<CallQueuedEvent>> SendInstruction(BaseAppInstruction instruction, CancellationToken cancellationToken = default(CancellationToken))
+        public Task<VoiceApiResult<CallQueuedEvent>> SendInstructionAsync(BaseAppInstruction instruction, CancellationToken cancellationToken = default)
```

Comment:
This method was already renamed to 'SendAsync' (hence the obsolete note). This method is only still here to not break things. Doesn't make sense to go renaming methods that only exist for backwards compatibility, as you now break there sole purpose ...